sqlmesh/core/config/scheduler.py

```diff
@@ -251,6 +251,60 @@ def get_client(self, console: t.Optional[Console] = None) -> AirflowClient:
         )
 
 
+class YCAirflowSchedulerConfig(_BaseAirflowSchedulerConfig, BaseConfig):
+    """The Yandex Cloud Managed Airflow Scheduler configuration.
+
+    Args:
+        airflow_url: The URL of the Airflow Webserver.
+        username: The Airflow username.
+        password: The Airflow password.
+        dag_run_poll_interval_secs: Determines how often a running DAG can be polled (in seconds).
+        dag_creation_poll_interval_secs: Determines how often SQLMesh should check whether a DAG has been created (in seconds).
+        dag_creation_max_retry_attempts: Determines the maximum number of attempts that SQLMesh will make while checking for
+            whether a DAG has been created.
+        backfill_concurrent_tasks: The number of concurrent tasks used for model backfilling during plan application.
+        ddl_concurrent_tasks: The number of concurrent tasks used for DDL operations (table / view creation, deletion, etc).
+        max_snapshot_ids_per_request: The maximum number of snapshot IDs that can be sent in a single HTTP GET request to the Airflow Webserver (Deprecated).
+        use_state_connection: Whether to use the `state_connection` configuration to access the SQLMesh state.
+        default_catalog_override: Overrides the default catalog value for this project. If specified, this value takes precedence
+            over the default catalog value set on the Airflow side.
+        token: The IAM-token for API authentification.
+    """
+
+    airflow_url: str
+    username: str
+    password: str
+    token: str
+    dag_run_poll_interval_secs: int = 10
+    dag_creation_poll_interval_secs: int = 30
+    dag_creation_max_retry_attempts: int = 10
+
+    backfill_concurrent_tasks: int = 4
+    ddl_concurrent_tasks: int = 4
+
+    max_snapshot_ids_per_request: t.Optional[int] = None
```

Comment:
This attribute is deprecated. Let's remove it from the new config.

Reply:
Is only `max_snapshot_ids_per_request` deprecated, or are all of them? I see that `backfill_concurrent_tasks` and `ddl_concurrent_tasks` are still in use, even though the validator has `check_field=False`: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/core/config/common.py#L49